.github/workflows/prod-wf.yml

```diff
@@ -1,5 +1,5 @@
-name: Lint and test
-on: [pull_request]
+name: Production workflow
+on: [push]
```

Comment:
This change reverts https://github.com/mystic-ai/pipeline/pull/203, which I don't think we want to do.